NeuroRuler/utils/img_helpers.py

```diff
@@ -9,6 +9,12 @@
 import NeuroRuler.utils.constants as constants
 
 
+class ImageProperties(NamedTuple):
+    center: tuple
```

Comment:
I don't think this is a valid type hint, you will need to have this import at the first line after the comments:

```
"""Image ....
"""

from __future__ import annotations

from typing import NamedTuple, Union
...
```

Otherwise it should be:

```
from typing import ..., Tuple

class ...:
    center: Tuple
```

But, perhaps more clearly:

```
class ...:
    #: Center of the image (x, y)
    center: tuple[float, float]

    #: Size of the image (width, height)
    size: tuple[float, float]

    #: Spacing of the image, spacing means ...
    spacing: tuple[float, float]

```

Reply:
Done in 4d45e12

Reply:
We should also add `from __future__ import annotations`, otherwise < 3.10 will return error:

```
test.py:5: error: Implicit generic "Any". Use "typing.Tuple" and specify generic parameters  [type-arg]
```